orders-accounting/src/error.rs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2024 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use common::{chain::OrderId, primitives::Amount};
+
+#[derive(thiserror::Error, Debug, PartialEq, Eq, Clone)]
+pub enum Error {
+    #[error("Accounting storage error")]
+    StorageError(#[from] chainstate_types::storage_result::Error),
+    #[error("Base accounting error: {0}")]
+    AccountingError(#[from] accounting::Error),
+    #[error("Order already exist: `{0}`")]
+    OrderAlreadyExists(OrderId),
+    #[error("Data for order {0}` not found")]
```

Comment:
Here and below you have a unmatched backtick in the error messages.
Also, `AccountingError`'s message (above) doesn't use backtick at all.